lib/src/view/game/game_body.dart

```diff
@@ -78,6 +80,7 @@ class GameBody extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final ctrlProvider = gameControllerProvider(id);
+    ref.watch(chatControllerProvider);
```

Comment:
I assume you needed an eager initialisation of the provider to ensure it could fetch the `full` event, this should not be needed anymore if we refactor as suggested above comment.